src/page-weekly-snapshot/LocaleStatsTable.tsx

```diff
@@ -0,0 +1,329 @@
+) {
+  return numFacilitiesLastWeek <= numFacilitiesThisWeek ? "+" : "-";
+}
+
+function getDay(nytData: NYTCountyRecord[] | NYTStateRecord[], day: number) {
+  if (day === 1) {
+    return minBy(nytData, (d: NYTCountyRecord | NYTStateRecord) => d.date);
+  } else {
+    return maxBy(nytData, (d: NYTCountyRecord | NYTStateRecord) => d.date);
+  }
+}
+
+function getCountyDataByName(counties: NYTCountyRecord[]) {
+  const dataByCounty: { [countyName: string]: NYTCountyRecord[] } = {};
+
+  counties.forEach((data: NYTCountyRecord) => {
+    if (dataByCounty[data.county]) {
+      dataByCounty[data.county] = [...dataByCounty[data.county], data];
+    } else {
+      dataByCounty[data.county] = [data];
+    }
+  });
+
+  return dataByCounty;
+}
+
+function calculatePerCapitaIncrease(
+  daySevenCases: number | undefined,
+  dayOneCases: number | undefined,
+  countyPopulation: number | undefined,
+) {
+  if (
+    (!daySevenCases && daySevenCases !== 0) ||
+    (!dayOneCases && dayOneCases !== 0) ||
+    !countyPopulation
+  )
+    return;
+  return (daySevenCases - dayOneCases) / countyPopulation;
+}
+
+function getCountyIncreasePerCapita(
+  counties: NYTCountyRecord[],
+  stateLocaleData: Map<string, LocaleRecord> | undefined,
+) {
+  const dataByCounty = getCountyDataByName(counties);
+  const countyCasesIncreasePerCapita: PerCapitaCountyCase[] = [];
+
+  for (const countyName in dataByCounty) {
+    const dayOne = getDay(dataByCounty[countyName], 1);
+    const daySeven = getDay(dataByCounty[countyName], 7);
+    const countyPopulation = stateLocaleData?.get(countyName)?.totalPopulation;
+    countyCasesIncreasePerCapita.push({
+      name: countyName,
+      casesIncreasePerCapita: calculatePerCapitaIncrease(
+        daySeven?.cases,
+        dayOne?.cases,
+        countyPopulation,
+      ),
+    });
+  }
+  return countyCasesIncreasePerCapita;
+}
+
+function getFacilitiesInCountiesToWatch(
+  facilities: Facility[],
+  countiesToWatch: string[],
+) {
+  let result = "";
+  for (let i = 0; i < facilities.length; i++) {
+    const modelInputs = facilities[i].modelInputs;
+    const facilityCounties = Object.values(pick(modelInputs, "countyName"));
+    for (let j = 0; j < facilityCounties.length; j++) {
+      const currCounty = facilityCounties[j];
+      if (currCounty && countiesToWatch.includes(currCounty)) {
+        result += facilities[i].name + ", ";
+      }
+    }
+  }
+  result = result.slice(0, -2);
+  return result;
+}
+
+function getFacilitiesCounties(facilities: Facility[]) {
+  let facilitiesCounties: string[] = [];
+  for (let i = 0; i < facilities.length; i++) {
+    const modelInputs = facilities[i].modelInputs;
+    const facilityCounties = Object.values(pick(modelInputs, "countyName"));
+    for (let j = 0; j < facilityCounties.length; j++) {
+      const currCounty = facilityCounties[j];
+      if (currCounty && !facilitiesCounties.includes(currCounty)) {
+        facilitiesCounties.push(currCounty);
+      }
+    }
+  }
```

Comment:
if I may suggest something more concise .... `facilitiesCounties = new Set(facilities.map(facility=> facility.modelInputs.countyName))`

this won't be equivalent if `facilityCounties` can ever have more than one item, same as above ... but unless I'm missing something, I don't think it can?